channels/us.m3u

```diff
@@ -1367,6 +1369,8 @@ https://cdn87.theus6tv.tk/locals/SanDiego/kpbs-dt2.m3u8
 https://cs.ebmcdn.net/eastbay-live-hs-1/apt/mp4:apt-world/playlist.m3u8
 #EXTINF:-1 tvg-id="PCCTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/9PEAPwY.png" group-title="Local",PCC-TV Pinellas County FL (480p)
 http://granicusliveus1-a.akamaihd.net/pinellas/G1551_004/playlist.m3u8
+#EXTINF:-1 tvg-id="PenthouseWorld.us" tvg-country="US" tvg-language="" tvg-logo="" group-title="XXX",Penthouse World
+http://45.5.8.78:8000/play/a02n
```

Comment:
infinite load

Reply:
Ah. I'll delete it if it doesn't change after some time.